docs/guides/1230-chain-actions.md

```diff
@@ -0,0 +1,54 @@
+---
+slug: /1230/chain-actions
+displayed_sidebar: europa
+---
+
+# Chaining actions
+
+Dependencies are materialized at runtime, when your Cue files are parsed and the corresponding DAG gets generated:
+
+```cue
+// Prerequisite action that runs when `test` is being called
+_dockerCLI: alpine.#Build & {
+    packages: {
+        bash: {}
+    }
```

Comment:
Could be directly `packages: bash: _`
And I think using `{}` isn't the standard, use `_` instead